diff --git a/node_modules/@ryanflorence/md/dist/index.js b/node_modules/@ryanflorence/md/dist/index.js
index 2c919e2..9bcc667 100644
--- a/node_modules/@ryanflorence/md/dist/index.js
+++ b/node_modules/@ryanflorence/md/dist/index.js
@@ -89,6 +89,7 @@ let remarkCodeBlocksShiki = () => {
                         "shellscript",
                         "html",
                         "css",
+                        "diff"
                     ],
                 }));
         let themeName = "base16";
